scripts/fix_lint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+# Copyright 2018-present Kensho Technologies, LLC.
```

Comment:
not sure that this matters at all, but how do we pick what year goes here?

Reply:
The year the file was added, and yes — it matters a lot. Happy to chat if you're interested in the details.

Reply:
No idea. I usually use a year between the first commit date and now :D Will update this one to 2020